### ISSUE TITLE
UI (a new look)

### ISSUE THREAD

**Issue by mleblanc321** [WheatonCS/Lexos#874]
Our UI is tired. It looks like we made ui decisions one at a time over multiple years using many different, often very novice, programmers. @Myles-Trevino has a new branch (dang ... gotta go find it):
feature/front-end-rewrite

* this does not use Bootstrap (omg!  but myles (and i) want to raise the issue that perhaps it is not our best move;
 - each summer, we intro bootstrap to ugrads who don't know bootstrap
 - myles' new look takes advantage of the modern way to work on lightweight GUIS: HTML5++ ... gone are the days where we *had* to rely on bootstrap and grids; for what we need, simple 5+css simplified (really!) may be enough
- even though Weiqi's new tokenizer is much improved, myles thinks we can also do this without datatables, bootstrap, etc;

@scottkleinman you are heavily invested in all this ... and we've waited until now to show you some new thoughts;  @chantisnake we know you are listening;

in short: this Issue may generate more discussion/comments/critique than we want to entertain in an Issue, but here it is (we look forward to your comments)

**Comment by mleblanc321** [comment]
feature/front-end-rewrite

**Comment by scottkleinman** [comment]
Definitely worth a lot of thought. I'll take a look at the new branch this weekend and try to contribute some feedback.

**Comment by mleblanc321** [comment]
status:  we will move forward with @Myles-Trevino's branch;  starting later this week, others will work in pairs and start a new branch off myles' and work on individual tools;  testing, tool tips, porting ITM to github;  this is a tough decision, but like each architectural move we've made over the last 6 years, my gut says this is also a good move ... more soon

**Comment by chantisnake** [comment]
Interesting, so we are not using any frontend libraries anymore? Just pure css and html?
I understand css has improved a lot in recent years, but keep in mind not every bowser supports these improvements.
